packages/react/spec/auto/hooks/useTable.spec.tsx

```diff
@@ -461,7 +461,7 @@ describe("useTable hook", () => {
       `);
     });
 
-    it.each(["password", "hasManyThrough"])("should throw an error if the selected field type is %s", async (fieldType) => {
+    it.each(["password"])("should throw an error if the selected field type is %s", async (fieldType) => {
```

Comment:
I think we might also want a test like the one above that confirms that the hasmanythrough is loading data correctly.

Reply:
Good call. Updated with some tests that show `hasManyThrough` in action with the `useTable` hook